frontend/src/pages/sites/sites.tsx

```diff
@@ -136,47 +136,49 @@ const Sites: React.FC = () => {
       <div className={styles['page-wrapper']}>
         {isSitesShow ? (
           <>
-            <div className={styles['button-wrapper']}>
-              <div>
-                <Link to={AppRoute.MY_PROJECTS}>
-                  <span className={styles['link-to-projects']}>
-                    <Icon
-                      iconName="arrowLeft"
-                      className={styles['back-icon']}
-                    />
-                  </span>
-                </Link>
+            <div>
+              <div className={styles['button-wrapper']}>
+                <div>
```

Comment:
```suggestion
              <div className={styles['button-wrapper']}>
```
remove useless wrappers